components/form.js

```diff
@@ -71,6 +71,19 @@ export default class Form extends React.Component {
       });
     }
   }
+  isOnlineEvent = e => {
+    let formDataArray = JSON.parse(
+      localStorage.getItem("Failed Form Submission Data")
+    );
+    if (!formDataArray) {
+      return;
+    } else {
+      formDataArray.map(formData => {
+        postFormData(formData, this.state.token.access_token);
+      });
+      localStorage.removeItem("Failed Form Submission Data");
```

Comment:
it's ok to do so in a separate PR, but you should consider how to only remove items if they were _successfully_. sent to the server and how to handle failures in items here. Maybe you can create a ticket for that ?

Reply:
Created a ticket__